### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by elvisish** [codecat/godot-tbloader#105] [General]
Could there be a new release for the latest version of TBLoader that includes default materials?

**Comment by codecat** [comment]
I have a sticky note to do this. Might look at it in the morning. Sorry for the delay.

**Reply by elvisish** [reply]
> I have a sticky note to do this. Might look at it in the morning. Sorry for the delay.

No problem, the release is the built extension isn't it? I don't think I can just download the source and use that if there's no built release?

**Reply by codecat** [reply]
You'll have to compile the extension yourself (which is not super difficult).

**Reply by elvisish** [reply]
> You'll have to compile the extension yourself (which is not super difficult).

Ahh I've never had luck with compiling, will you be able to make a release soon?

**Reply by codecat** [reply]
Soon yeah. I don't have easy access to all the devices I use for building right away (Mac & Linux), but I'll do my best to get this sorted soon 👍
